docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -0,0 +1,141 @@
+---
+lang: en
+title: 'Add Model Relations'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-relations.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Define the model relation
+
+We are going to add the model relation to indicate the relation that `TodoList`
+_hasMany_ `Todo` using the `lb4 relation` command.
+
+```sh
+$ lb4 relation
+? Please select the relation type hasMany
+? Please select source model TodoList
+? Please select target model Todo
+? Foreign key name to define on the target model todoListId
+? Source property name for the relation getter todos
+? Allow TodoList queries to include data from related Todo instances? Yes
+   create src/controllers/todo-list-todo.controller.ts
+
+Relation HasMany was created in src/
+```
+
+Now, we're going to add the relation for `Todo`. That is, `Todo` _belongsTo_
+`TodoList`.
+
+```sh
+$ lb4 relation
+? Please select the relation type belongsTo
+? Please select source model Todo
+? Please select target model TodoList
+? Source property name for the relation getter todoListId
+? Allow Todo queries to include data from related TodoList instances? Yes
+   create src/controllers/todo-todo-list.controller.ts
+
+Relation BelongsTo was created in src/
+```
+
+### Behind the scene
+
+If you want to understand the code changes introduced from the relation
+generator command, read on the details in this section; otherwise, you are ready
+to move to the next step to create the controller.
+
+#### Relation decorators
+
+When specifying the `hasMany` relation, it adds `@hasMany()` decorator defines
```

Comment:
> When specifying the `hasMany` relation, it adds `@hasMany()` decorator defines this property.

What are we trying to say here?

Reply:
trying to say when running the `lb4 relation` command, the decorator was added to the property. Let me do some rewording here.